app/Http/Controllers/BookController.php

```diff
@@ -171,6 +184,7 @@ public function update(Request $request, string $slug)
             'description' => ['string', 'max:1000'],
             'image'       => array_merge(['nullable'], $this->getImageValidationRules()),
             'tags'        => ['array'],
+            'default_template'  => ['nullable', 'exists:pages,id'], 
```

Comment:
Currently, if the template option is removed when editing a book the column value will end up as `0`. Since nulls are supported in this column, i think the value should end up `null` instead.